MobileSyncExplorerReactNative/package.json

```diff
@@ -7,17 +7,17 @@
         "test": "jest"
     },
     "sdkDependencies": {
-        "SalesforceMobileSDK-iOS": "https://github.com/forcedotcom/SalesforceMobileSDK-iOS.git#dev",
-        "SalesforceMobileSDK-Android": "https://github.com/forcedotcom/SalesforceMobileSDK-Android.git#dev"
+        "SalesforceMobileSDK-Android": "https://github.com/wmathurin/SalesforceMobileSDK-Android.git#rn0701",
+        "SalesforceMobileSDK-iOS": "https://github.com/forcedotcom/SalesforceMobileSDK-iOS.git#dev"
     },
     "dependencies": {
-        "@react-native-community/masked-view": "^0.1.10",
+        "@react-native-masked-view/masked-view": "^0.2.7",
         "@react-navigation/native": "^6.0.2",
         "@react-navigation/stack": "^6.0.7",
-        "react": "17.0.2",
-        "react-native": "0.67.1",
-        "react-native-force": "git+https://github.com/forcedotcom/SalesforceMobileSDK-ReactNative.git#dev",
         "react-native-elements": "^3.4.2",
+        "react": "18.1.0",
+        "react-native": "0.70.1",
+        "react-native-force": "git+https://github.com/wmathurin/SalesforceMobileSDK-ReactNative.git#rn0701",
```

Comment:
Pointing to `wmathurin/SalesforceMobileSDK-ReactNative.git#rn0701` for now allow CI to succeed.
Will point to `forcedotcom/SalesforceMobileSDK-ReactNative.git#dev` before merging